scrapers/ak/bills.py

```diff
@@ -120,21 +140,20 @@ def scrape_bill(
                     if sponsor:
                         bill.add_sponsorship(
                             sponsor,
+                            chamber=sponsor_chamber,
                             entity_type="person",
                             classification="cosponsor",
                             primary=False,
                         )
 
-            else:
+            if sponsors_cnt == 0:
                 # Committee sponsorship
                 spons_str = spons_str.strip()
-
                 if re.match(r" BY REQUEST OF THE GOVERNOR$", spons_str):
                     spons_str = re.sub(
                         r" BY REQUEST OF THE GOVERNOR$", "", spons_str
                     ).title()
                     spons_str = spons_str + " Committee (by request of the governor)"
-
                 if spons_str:
                     bill.add_sponsorship(
```

Comment:
Can you please add logic here that adds the `chamber` argument for bills sponsored by an organization? It looks like every organization name starts with either `SENATE` or `HOUSE`, so if `SENATE` then `chamber=upper` and if `HOUSE` then `chamber=lower`